src/shadowbox/integration_test/test.sh

```diff
@@ -90,6 +90,8 @@ function cleanup() {
   # Sets everything up
   export SB_API_PREFIX=TestApiPrefix
   SB_API_URL=https://shadowbox/${SB_API_PREFIX}
+  export TMP_STATE_DIR=$(mktemp -d)
```

Comment:
Maybe delete the temp directory on `cleanup()` if DEBUG is disabled?

Reply:
Done